.changeset/pre.json

```diff
@@ -0,0 +1,36 @@
+{
+  "mode": "pre",
+  "tag": "beta",
+  "initialVersions": {
+    "example-nextjs": "0.1.0",
+    "example-node": "0.0.1",
+    "example-react-native": "0.0.1",
+    "example-shared": "0.1.0",
+    "example-wagmi": "0.1.0",
+    "@lens-protocol/api-bindings": "0.6.0",
+    "@lens-protocol/blockchain-bindings": "0.6.0",
+    "@lens-protocol/client": "0.3.0",
+    "@lens-protocol/domain": "0.6.0",
+    "@lens-protocol/eslint-config": "0.5.0",
+    "@lens-protocol/gated-content": "0.1.0",
+    "@lens-protocol/prettier-config": "0.5.0",
+    "@lens-protocol/react": "0.6.0",
+    "@lens-protocol/react-web": "0.6.0",
+    "@lens-protocol/sdk-gated": "1.2.0",
+    "@lens-protocol/shared-kernel": "0.6.0",
+    "@lens-protocol/storage": "0.6.0",
+    "@lens-protocol/tsconfig": "0.1.0",
+    "@lens-protocol/wagmi": "0.6.0"
+  },
+  "changesets": [
```

Comment:
what is this file for? haven't seen that before

Reply:
Changeset feature: https://github.com/changesets/changesets/blob/main/docs/prereleases.md